docs/DeveloperGuide.adoc

```diff
@@ -795,19 +1014,84 @@ Private contact detail
 A contact detail that is not meant to be shared with others
 ....
 
+[[critical]]
+Critical
+
+....
+Requiring immediate attention
+....
+
+[[blacklist]]
+Blacklist
+
+....
+A list of people who are banned from borrowing
+....
+
+[[bookmark]]
+Bookmark
+
+....
+An action that enables certain contacts to be displayed on application start-up
+....
+
+
 [appendix]
 == Product Survey
 
-*Product Name*
+*Address Book*
```

Comment:
Would it be neater to display the pros and cons from your product survey in a table?

Reply:
Noted. Necessary changes have been made by the group.